docs/community-libraries-and-navigators.md

```diff
@@ -64,15 +64,33 @@ Provides simple HOCs that map react-navigation props to your screen components d
 
 This component implements [iOS large header with grow/shrink on scroll](https://react-navigation.canny.io/feature-requests/p/ios-11-large-header-and-growshrink-on-scroll), made by [@jonsamp](https://github.com/jonsamp). Note that it doesn't handle header animation between screens, it only handles animating the header title on scroll.
 
-To use this component for header in React Navigation, we need to disable the default header:
+headerMode is an option in createStackNavigator's config, not in navigationOptions. I'd document both, headerMode: 'none' to disable globally, and header: null to disable header for one screen.
```

Comment:
Hmm. I was replying to you, not suggesting to add this text to the README.

Reply:
oh oops totally my bad